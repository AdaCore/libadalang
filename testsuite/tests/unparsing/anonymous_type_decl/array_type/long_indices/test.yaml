description: |
    This test checks the formatting of an anonymous array type declaration with
    multiple indices.
    It is expected the declaration breaks before `of` keywords and the new
    line to have the same indentation as the `array` keyword.
    Is is also expected that the indices have continuation line indentation
    relative to where the `array` keyword ends.

driver: unparser
rule: anonymous_type_decl
