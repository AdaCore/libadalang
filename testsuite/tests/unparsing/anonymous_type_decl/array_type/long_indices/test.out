array (Buffer_Index_1,
       Buffer_Index_2,
       Buffer_Index_3,
       Buffer_Index_4,
       Buffer_Index_5)
of Character
     .Long_Dotted_Name
     .Long_Dotted_Name
     .Long_Dotted_Name
     .Long_Dotted_Name
